css/src/components/table.css

```diff
@@ -0,0 +1,168 @@
+.yoast table {
+	font-size: var(--font-size-default);
+	color: var(--color-default);
+	line-height: 1.2;
+	width: 100%;
+	border-bottom: var(--border-default);
+	border-spacing: 0;
+}
+
+.yoast tfoot th {
+	border-bottom: none;
+}
+
+.yoast th {
+	border-bottom: var(--border-default);
+}
+
+.yoast th, .yoast th.sortable {
+	color: var(--color-dark);
+	text-align: left;
+	padding: 16px;
+}
+
+.yoast th.sortable a, .yoast th a {
+	color: var(--color-dark);
+	padding: 0;
+	font-weight: 500;
+}
+
+.yoast tr {
+	height: 50px;
+}
+
+.yoast td {
+	padding: 16px;
+	border-bottom: var(--border-default);
+	font-size: var(--font-size-default);
+}
+
+.yoast .tablenav {
+	height: auto;
+	margin: 6px 0 16px;
+}
+
+.yoast .tablenav.top .tablenav-pages {
+	display: none;
+}
+
+.yoast .tablenav.top #posts-filter .tablenav-pages {
+	display: block;
+}
+
+.yoast .tablenav .yoast-button {
+	color: var(--color-dark);
+}
+
+.yoast .tablenav .search-box{
+	float: right;
+}
+
+.yoast .tablenav .search-box div {
+	float: left;
+	width: 200px;
+	margin-right: 8px;
+}
+
+.yoast .tablenav .displaying-num {
+	margin-right: 22px;
+}
+
+.yoast .tablenav .paging-input {
+	margin-right: 6px;
+}
+
+.yoast .column-actions {
+	text-align: right;
+	width: 200px;
+}
+
+.yoast .tablenav .tablenav-pages .button,
+.yoast .tablenav .tablenav-pages .button.disabled,
+.yoast .tablenav .tablenav-pages .tablenav-pages-navspan {
+	display: inline-block;
+	position: relative;
+	text-decoration: none;
+	border-radius: 4px;
+	transition: background-color 150ms ease-out 0s;
+	cursor: pointer;
+	padding: 10px 13px 12px;
+	border: 1px solid rgba(0,0,0, 0.2)!important;
```

Comment:
- rgba values should be separated by a space (also inconsistent with the syntax in the rgba value on the line below
- we shouldn't use `!important` but if it's really, really, necessary then it needs a space before the exclamation mark